server/cluster/shard.go

```diff
@@ -21,3 +24,68 @@ type ShardTablesWithRole struct {
 	tables    []*Table
 	version   uint64
 }
+
+// Shard FSM Const Definition
+// It contains the event name and the state name
+const (
+	EventTransferLeader          = "TransferLeader"
+	EventTransferFollowerFailed  = "TransferFollowerFailed"
+	EventTransferFollower        = "TransferFollower"
+	EventTransferLeaderFailed    = "TransferLeaderFailed"
+	EventPrepareTransferFollower = "PrepareTransferFollower"
+	EventPrepareTransferLeader   = "PrepareTransferLeader"
+	StateLeader                  = "LEADER"
+	StateFollower                = "FOLLOWER"
+	StatePendingLeader           = "PENDING_LEADER"
+	StatePendingFollower         = "PENDING_FOLLOWER"
+)
+
+// Declare the source state array of FSM, avoid creating arrays repeatedly every time you create an FSM
+var (
+	EventTransferLeaderSrc          = []string{StatePendingLeader}
+	EventTransferFollowerFailedSrc  = []string{StatePendingFollower}
+	EventTransferFollowerSrc        = []string{StatePendingFollower}
+	EventTransferLeaderFailedSrc    = []string{StatePendingLeader}
+	EventPrepareTransferFollowerSrc = []string{StateLeader}
+	EventPrepareTransferLeaderSrc   = []string{StateFollower}
```

Comment:
These variable seems no need to be exported?

Reply:
fix